packages/neuron-ui/src/types/Controller/index.d.ts

```diff
@@ -35,7 +35,12 @@ declare namespace Controller {
     newPassword?: string
     name?: string
     device?: any
+  }
+
+  interface UpdateWalletStartBlockNumberParams {
+    id: string
     startBlockNumber?: string
+    clearAllData?: boolean
```

Comment:
Maybe `clearAllData` can be avoided by comparing `new block number` and `previously set block number` in `neuron-wallet`

Reply:
I guess it's better to clear all data when starting a smaller block number, because some addresses may not be derived with a bigger block number in the previous.

Reply:
Sure, but my point is that the `clear all data` is not necessary in the API, it can be defined in the service, instead of exposing a field for other callers, because `if data should be removed` is determined by the service, not an external caller.

The logic should be `if the new start point is earlier than the previous one, then remove data` instead of `if the caller asks to remove data, then remove data`